src/metabase/models/setting.clj

```diff
@@ -148,9 +148,54 @@
   [_setting]
   [:key])
 
+(def ^:private exported-settings
+  '#{admin-email
+     application-colors
+     application-favicon-url
+     application-font
+     application-font-files
+     application-logo-url
+     application-name
+     available-fonts
+     available-locales
+     available-timezones
+     breakout-bins-num
+     custom-formatting
+     custom-geojson
+     custom-geojson-enabled
+     enable-content-management?
+     enable-embedding
+     enable-nested-queries
+     enable-sandboxes?
+     enable-whitelabeling?
+     enable-xrays
+     hide-embed-branding?
+     humanization-strategy
+     landing-page
+     loading-message
+     map-tile-server-url
+     max-results-bare-rows
+     native-query-autocomplete-match-style
+     persisted-models-enabled
```

Comment:
This is another one that seems more likely be different between dev and prod than others.